packages/design-system/src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,344 @@
+    }
+
+    /**
+     * Renders all Page components in range (3 pages) to Pagination component.
+     */
+
+    pageRange.map((page) => {
+      pages.push(
+        <Page
+          renderPageHref={renderHref(page)}
+          key={`page-${page}`}
+          index={page}
+          isActive={currentPage === page}
+          onPageChange={pageChange(page)}
+        />
+      );
+    });
+
+    /**
+     * Defines if/when the Ellipses component appears
+     * at the end of the Pagination component -
+     * as long as there are fewer than 7 pages.
+     */
+    if (currentPage <= totalPages - 3 && totalPages > maxVisiblePages) {
+      if (currentPage < totalPages - 3) {
+        pages.push(<Ellipses key="ellipses-2" />);
+      }
+
+      pages.push(
+        <Page
+          renderPageHref={renderHref(totalPages)}
+          key={`page-${totalPages}`}
+          index={totalPages}
+          isActive={currentPage === totalPages}
+          onPageChange={pageChange(totalPages)}
+        />
+      );
+    }
+  }
+
+  const startIcon = (
+    <svg
+      aria-hidden="true"
+      focusable="false"
+      data-prefix="fas"
+      data-icon="chevron-left"
+      className="ds-c-pagination__nav--image"
+      role="img"
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 320 512"
+    >
+      <path
+        fill="currentColor"
+        d="M34.52 239.03L228.87 44.69c9.37-9.37 24.57-9.37 33.94 0l22.67 22.67c9.36 9.36 9.37 24.52.04 33.9L131.49 256l154.02 154.75c9.34 9.38 9.32 24.54-.04 33.9l-22.67 22.67c-9.37 9.37-24.57 9.37-33.94 0L34.52 272.97c-9.37-9.37-9.37-24.57 0-33.94z"
+      />
+    </svg>
+  );
+
+  const endIcon = (
+    <svg
+      aria-hidden="true"
+      focusable="false"
+      data-prefix="fas"
+      data-icon="chevron-right"
+      className="ds-c-pagination__nav--image"
+      role="img"
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 320 512"
+    >
+      <path
+        fill="currentColor"
+        d="M285.476 272.971L91.132 467.314c-9.373 9.373-24.569 9.373-33.941 0l-22.667-22.667c-9.357-9.357-9.375-24.522-.04-33.901L188.505 256 34.484 101.255c-9.335-9.379-9.317-24.544.04-33.901l22.667-22.667c9.373-9.373 24.569-9.373 33.941 0L285.475 239.03c9.373 9.372 9.373 24.568.001 33.941z"
+      />
+    </svg>
+  );
+
+  return (
+    <nav className={classes} aria-label={ariaLabel} {...rest}>
+      {currentPage === 1 ? (
+        <span className="ds-c-pagination__nav ds-c-pagination__nav--disabled">
+          <span className="ds-c-pagination__nav--img-container ds-c-pagination__nav--img-container-previous">
+            {startIcon}
+          </span>
+          {startLabelText}
+        </span>
+      ) : (
+        <Button
+          variation="transparent"
+          href={renderHref(currentPage - 1)}
+          onClick={pageChange(currentPage - 1)}
+          aria-label={startAriaLabel}
+          style={{ visibility: currentPage === 1 ? 'hidden' : 'visible' }}
+          aria-hidden={currentPage === 1}
```

Comment:
[question] are these two lines still needed? If not, consider removing

Reply:
you are correct! removing!